### PR TITLE
try to download generic wheels if they exist

### PR DESCRIPTION
**Pull request by mboisson**

### CODE REVIEW

File: build_wheel.sh [hunk 1]
```diff
@@ -283,6 +283,18 @@ function download()
 			ARCHNAME=$(PIP_CONFIG_FILE= eval $PACKAGE_DOWNLOAD_CMD |& tee download.log | grep "Saved " | awk '{print $2}')
 		fi
 	fi
+	if [[ -z "$ARCHNAME" ]]; then
+		echo "Trying to download without --no-binary to see if it is a py3-none-any wheel."
+		PACKAGE_DOWNLOAD_CMD=${PACKAGE_DOWNLOAD_CMD//--no-binary \$PACKAGE/}
+		PACKAGE_DOWNLOAD_CMD=${PACKAGE_DOWNLOAD_CMD//--no-use-pep517/}
+		ARCHNAME=$(PIP_CONFIG_FILE= eval $PACKAGE_DOWNLOAD_CMD |& tee download.log | grep "Saved " | awk '{print $2}')
+		if [[ $ARCHNAME =~ .*-py3-none-any.* ]]; then
```
Comment (ccoulombe) [inline build_wheel.sh:291]: This generates an error:
```
+ ARCHNAME=./tensorboardX-2.6-py2.py3-none-any.whl
+ '[[ ./tensorboardX-2.6-py2.py3-none-any.whl' '=~' '.*-py3-none-any.*' ']]'
build_wheel.sh: line 291: [[ ./tensorboardX-2.6-py2.py3-none-any.whl: No such file or directory
+ unset ./tensorboardX-2.6-py2.py3-none-any.whl
build_wheel.sh: line 294: unset: `./tensorboardX-2.6-py2.py3-none-any.whl': not a valid identifier
```

**Review by ccoulombe — CHANGES_REQUESTED** [review]

**Comment by ccoulombe** [comment]
I also want to further test it before merging it

File: build_wheel.sh (hunk 1 above)
Comment (mboisson) [reply build_wheel.sh]: The error is fixed. The if-else can't be removed, as we do want to build non-py3 wheels.

**Review by ccoulombe — APPROVED** [review]
:+1:

**Pull request merged**